index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        a {
+            font-family: Verdana, Geneva, Tahoma, sans-serif;
+        }
+        h1 {
+            color:lightseagreen;
+            display: flex;
+            justify-content: center;
+            font-size: 75px;
+        }
+
+        div, ul {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+            padding: 0;
+        }
+
+        li {
+            display: flex;
+            list-style-type: none;
+            font-size: 50px;
+            border: 1px solid black;
+            background-color: lightgray;
+            height: 75px;
+            width: 75px;
+            border-radius: 50%;
+            justify-content: center;
+            align-items: center;
+            margin: 1rem;
+            }
+        
+        button {
+            height: 50px;
+            width: 100px;
+            font-weight: bold;
+            margin: 1rem;
+        } 
+
+    </style>
+</head>
+<body>
+    <h1>Acme Numbers and Events</h1>
+    <div>
+        <button id = 'add'>Add Number</button>
+        <button id = 'clear'>Clear</button>
+        <h3>The average of all the numbers is 0</h3>
+    </div>
+    <ul>
+    </ul>
+    <script>
+        const addButton = document.querySelector('#add')
+        const clearButton = document.querySelector('#clear')
+        const ul = document.querySelector('ul')
+        const h3 = document.querySelector('h3')
+        let sum = 0
+        let count = 0
+
+        function addNumber () {
+            const number = Math.floor(Math.random()*(100-1) + 1)
+            const li = document.createElement('li')
+            li.innerHTML = number
+            ul.appendChild(li)
+            sum += number
+            count ++
+            h3.innerHTML = `The average of all the numbers is ${Math.round(sum/count)}`
+        }
+
+        function clearNumber () {
+            ul.innerHTML = ''
+            sum = 0
+            count = 0
+            h3.innerHTML = `The average of all the numbers is 0`
+        }
+
+        addButton.addEventListener('click', function () {
+            addNumber()
+        })
```

Comment:
you can actually use a shorthand here- 
```suggestion
        addButton.addEventListener('click', addNumber)
```
this is because the second argument of `addEventListener` is a callback function- which means it's just a function that you don't invoke yet. javascript automatically invokes once the button is clicked